api/tests/core/mails/transactional/users/recredit_birthday_anniversary_to_underage_beneficiary_test.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+import pytest
+
+import pcapi.core.mails.testing as mails_testing
+from pcapi.core.mails.transactional.users.recredit_birthday_anniversary_to_underage_beneficiary import (
+    get_recredit_birthday_anniversary_to_underage_beneficiary_email_data,
+)
+from pcapi.core.mails.transactional.users.recredit_birthday_anniversary_to_underage_beneficiary import (
+    send_recredit_birthday_anniversary_email_to_underage_beneficiary,
+)
+from pcapi.core.payments import models as payments_models
+import pcapi.core.payments.conf as deposit_conf
+from pcapi.core.testing import override_features
+import pcapi.core.users.factories as users_factories
+from pcapi.models import db
+from pcapi.repository import transaction
+
+
+pytestmark = pytest.mark.usefixtures("db_session")
+
+
+class SendinblueSendNewlyEligibleUserEmailTest:
+    @override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=True)
+    def test_send_recredit_birthday_anniversary_email_to_underage_beneficiary(self):
+        # given
+        user = users_factories.UnderageBeneficiaryFactory(
+            dateOfBirth=(datetime.now() - relativedelta(years=16, days=5)), departementCode="93"
```

Comment:
Je ne pense pas qu'on ait besoin de spécifier le département, non ?

```suggestion
            dateOfBirth=(datetime.now() - relativedelta(years=16, days=5))
```

Idem pour le test plus bas.

Reply:
ok retiré